### PR TITLE
Copy-edit Power.md section

### PR DESCRIPTION
**Pull request by lproven**
I've added a bit more explanatory material from my own personal experience, including how to add a new swap partition and test it which was not covered. I linked to the Live DVD images and to the old hibernation chapter on the openSUSE wiki.

I am leaving SUSE today, sadly, so any comments or requests should be addressed by email to lproven@gmail.com as in an hour or 2 I won't be able to see my SUSE email any more...

### CODE REVIEW

File: project/docs/alternative_procurement.md [hunk 1]
```diff
@@ -4,80 +4,99 @@
...
-* _Ubuntu snaps_, which you install from the `snapd` command line utility, providing you (just like flatpaks) containerized applications with their dependencies
-* _AppImages_, which you manually download from websites and various hosts, providing you with a single executable file which your system will treat as an application, much like what you would see in a macOS environment
+* the official and unofficial repositories, which you interact with from `zypper`, and which provide binaries packaged as RPM files.
+* third-party providers, which you interact with from a command prompt by downloading files (for example, with `wget`, `curl`, or `git`). These may provide you with a something ready to install, complete with binaries and installation scripts, or may include just source code and build scripts, so you must compile them yourself.
+* _Flatpak_ is the GNOME Project's containerized application delivery tool. You typically interact with it using the `flatpak` command, and it provides apps packaged up with all their dependencies.
```
[the remark this answers is not in the document]
Comment (sknorr) [reply project/docs/alternative_procurement.md:9]: Mmh, Flatpak does have its origins in the GNOME project but I agree about the removal.

```suggestion
* _Flatpak_ is a containerized application delivery tool. You typically interact with it using the `flatpak` command, and it provides apps packaged up with all their dependencies.
```

File: project/docs/alternative_procurement.md [hunk 2]
```diff
@@ -4,80 +4,99 @@
...
+> Use `zypper lr -d` to list all your repositories and their addresses.
 2. Install the target package with `sudo zypper in <package_name>`. In certain cases you won't know the exact name of the package, so after adding the corresponding repository you can simply do `sudo zypper dup` to make sure that the package is installed among all the pending updates.
 
+## The _openSUSE Software_ website
+
+The openSUSE project maintains a website where contributors can host software packages for easy installation, especially those built using SUSE's _Open Build Service_. The site can be found at:
+
+[openSUSE Software](https://software.opensuse.org/) — https://software.opensuse.org
+
+It offers a search engine and easy one-click installation using the openSUSE _YaST_ administration tool, but it also offers packages for multiple other distributions at the discretion of the maintainer. You can find packages for Debian, Ubuntu, Fedora and other distributions as well as openSUSE.
```
[the remark this answers is not in the document]
Comment (sknorr) [reply project/docs/alternative_procurement.md:33]: I struck the one-click thing. And made it less wordy.

```suggestion
It offers a search engine and installation on openSUSE using the YaST administration tool. In addition, in some cases, the tool also offers downloadable packages for Debian, Ubuntu, Fedora, or other Linux distributions.
```

File: project/docs/alternative_procurement.md [hunk 3]
```diff
@@ -218,30 +238,67 @@ __Go for it when__:
 
 ## AppImages
 
-!!! warning
-    Please help finish this section.
+AppImage is the simplest cross-platform application package format, but that means that you need to do a little more work.
+
+Installing an AppImage is as simple as downloading it from the project's website and then marking it as executable. You might wish to put it in its own folder, or have a dedicated folder for AppImages. For instance, if you made such a folder in your home directory:
+
+    mkdir ~/appimage
```
[the remark this answers is not in the document]
Comment (sknorr) [reply project/docs/alternative_procurement.md:245]: Yech. Capitalized dir names? I know why I am not using AppImages. :)

```suggestion
    mkdir ~/Applications
```

**Comment by sknorr** [comment]
If there are no objections, I would like to close this one in favor of its replacement, #73.

**Comment by why-not-try-calmer** [comment]
Thanks! Will follow up

**Pull request closed without merging**